src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage {
+    public static final Map<LocalDate, Map<String, Integer>> fruitStorage = new HashMap<>();
```

Comment:
у тебе ж є моделька Fruit, давай зберігати її`
```suggestion
    public static final Map<LocalDate, Map<Fruit, Integer>> fruitStorage = new HashMap<>();
```